docs/src/configuration/app/timezone.md

```diff
@@ -10,13 +10,23 @@ That is generally fine but sometimes it's necessary to run cron tasks in a diffe
 
 ## Setting the system timezone for cron tasks
 
-The `timezone` property sets the timezone for which the `spec` property of any [cron tasks](/configuration/app/cron.md) defined by the application will be interpreted.  Its value is one of the [tz database region codes](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones) such as `Europe/Paris` or `America/New_York`.  This key will apply to all cron tasks defined in that file.
-
+The `timezone` property sets the timezone for which the `spec` property of any [cron tasks](/configuration/app/cron.md) defined by the application will be interpreted.
+It sets **when / in which timezone** your cron is run (to avoid having to calculate to and from UTC).
+Its value is one of the [tz database region codes](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones) such as `Europe/Paris` or `America/New_York`.  This key will apply to all cron tasks defined in that file.
 This entry is only meaningful on cron specs that specify a particular time of day, rather than a "time past each hour".  For example, `25 1 * * *` would run every day at 1:25 am in the timezone specified.
 
+
 ## Setting an application runtime timezone
 
-The application runtime timezone can also be set, although the mechanism varies a bit by the runtime.
+This is **how** the set application runtime interprets the timezone.
+The following lines in the `.platform.app.yaml` will allow your php runtime to use the `Europe/Berlin` timezone:
+```
+variables:
+   php:
+       "date.timezone": "Europe/Berlin"
+```
```

Comment:
A PHP example should be listed under the PHP bullet point, not privileged here.  We have a surprising amount of Node and Python code running, it seems. :-)

Also, I have no idea why the language list below is indented.  It shouldn't be.